src/Workspaces/Core/Portable/Workspace/ProjectSystem/ProjectSystemProjectOptionsProcessor.cs

```diff
@@ -71,7 +71,6 @@ private bool ReparseCommandLineIfChanged_NoLock(ImmutableArray<string> arguments
         // Dispose the existing stored command-line and then persist the new one so we can
         // recover it later.  Only bother persisting things if we have a non-empty string.
 
-        _commandLineStorage?.Dispose();
```

Comment:
this is hte one place we would actuallly dispose these streams.  however, it was for the ProjSysOptionProcessor.  But hte data here basically changes extraordinarily rarely (only when you change the references for a project).  So just allowing gc/finalization handle this is totally fine.